tests/test_dataset.py

```diff
@@ -1849,3 +1852,64 @@ def test_read_only_view() -> None:
         v_read.write(data=new_data)
 
     v_write.write(data=new_data)
+
+
+@pytest.fixture(
+    params=[
+        WKDataset,
+        TiffDataset,
+        TiledTiffDataset,
+    ]
+)
+def create_dataset(request: Any) -> Generator[MagDataset, None, None]:
+    dataset_type = request.param
+    with tempfile.TemporaryDirectory() as temp_dir:
+        if dataset_type == TiledTiffDataset:
+            ds = dataset_type.create(temp_dir, scale=(1, 1, 1), tile_size=(64, 64))
+        else:
+            ds = dataset_type.create(temp_dir, scale=(1, 1, 1))
```

Comment:
Could you use a non-trivial scale here, please? Since the returned bounding boxes should be in voxel-space, it shouldn't make a difference (but non-trivial scales make this test stronger).